### ISSUE TITLE
Broken CSS shadow in Admin -> User list search box

### ISSUE THREAD

**Issue by thorsteneckel** [zammad/zammad#1210]
### Infos:

* Used Zammad version: latest development
* Used Zammad installation source: source
* Operating system: macOS 10.12.4
* Browser + version: Chrome latest

The Admin -> User List view search box has a broken shadow.

<img width="395" alt="bildschirmfoto 2017-06-26 um 16 49 02" src="https://user-images.githubusercontent.com/3873515/27580089-ede9d08a-5b29-11e7-9819-e2f4901c2e28.png">

**Comment by mrflix** [comment]
Can't reproduce the visual bug. Do you have custom CSS running or chrome extensions that might influence the visual appearance of input fields?

**Comment by thorsteneckel** [comment]
Nope no extensions. I can reproduce it in my Chrome private mode with all extensions disabled:

<img width="1000" alt="bildschirmfoto 2017-07-17 um 13 09 18" src="https://user-images.githubusercontent.com/3873515/28265698-5b486d36-6af1-11e7-9425-979a2f71c4a7.png">

**Comment by mrflix** [comment]
Based on knowledge from this bug report: https://github.com/palantir/blueprint/pull/13
Apparently this problem only appears in Chrome on high-res screens.

**Comment by martini** [comment]
One addition: For mix it's only sometimes. It goes after scaling in/out with browser zoom level. So it seems that it is no CSS issue -> for me it's more a render issue of browser.

**Comment by mrflix** [comment]
Yeah as I said it appears to be a render issue in Google Chrome. But adding `will-change: transform` makes Chrome put the element onto a separate layer to make sure that the outline is visible.
